build/zeromq.js.patch

```diff
@@ -516,3 +552,54 @@ index 693d914..955dc2b 100644
    if (process.env.ZMQ_DRAFT === "true") {
      console.log("Enabling draft support")
      build_options += " -DENABLE_DRAFTS=ON"
+
+@@ -82,7 +87,7 @@ function main() {
+   console.log(cmake_configure)
+   exec(cmake_configure, execOptions)
+
+-  const cmake_build = `cmake --build ./build --config ${CMAKE_BUILD_TYPE} --target install`
++  const cmake_build = `cmake --build ./build --config ${CMAKE_BUILD_TYPE} --target install --component sdk`
+   console.log(cmake_build)
+   exec(cmake_build, execOptions)
+
+diff --git a/binding.gyp b/binding.gyp
+index 280b468..64e6a7e 100644
+--- a/binding.gyp
++++ b/binding.gyp
+@@ -154,7 +154,7 @@
+                       # 1 - MultiThreadedDebug (/MTd)
+                       # 2 - MultiThreadedDLL (/MD)
+                       # 3 - MultiThreadedDebugDLL (/MDd)
+-                      'RuntimeLibrary': 3,
++                      'RuntimeLibrary': 1,
+                       'AdditionalOptions': [
+                         '-std:c++17',
+                         "/DEBUG",
+@@ -171,7 +171,7 @@
+                     # # C:/Program Files (x86)/Microsoft Visual Studio/2019/Preview/VC/Tools/MSVC/14.29.29917/bin/Hostx64/x64/
+                     'VCCLCompilerTool': {
+                       'ExceptionHandling': 2,       # /EHsc
+-                      'RuntimeLibrary': 3,
++                      'RuntimeLibrary': 1,
+                       "DebugInformationFormat": "ProgramDatabase", # /Zi
+                       'AdditionalOptions': [
+                         '-std:c++17',
+@@ -243,7 +243,7 @@
+                   # 1 - MultiThreadedDebug (/MTd)
+                   # 2 - MultiThreadedDLL (/MD)
+                   # 3 - MultiThreadedDebugDLL (/MDd)
+-                  'RuntimeLibrary': 2,
++                  'RuntimeLibrary': 0,
+                   'AdditionalOptions': [
+                     '-std:c++17',
+                     '/EHsc'
+
+@@ -1736,7 +1743,7 @@ if(BUILD_SHARED OR BUILD_STATIC)
+           DESTINATION ${ZEROMQ_CMAKECONFIG_INSTALL_DIR})
+ endif()
+
+-option(ENABLE_CPACK "Enables cpack rules" ON)
++option(ENABLE_CPACK "Enables cpack rules" OFF)
```

Comment:
Good find, this is a public option, you can instead do `-DENABLE_CPACK=OFF` in build.ts